pkgs/tools/admin/afterburn/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, openssl, pkg-config, fetchCrate, rustPlatform }:
+
+(rustPlatform.buildRustPackage rec {
```

Comment:
We need to avoid the use of `buildRustPackage` in Nixpkgs. It abuses fixed-output derivations and may stop working in the future (https://github.com/NixOS/nix/issues/2270).

Reply:
Is there some kind of   cargo2nix tool I can use instead?

Reply:
There is carnix, which has different problems.

Reply:
TenX has a cargo2nix tool https://github.com/tenx-tech/cargo2nix

Reply:
@edolstra now that Nix also uses some rust, do you have some new insights in how you would like to see these kind of things one in a reproducible manner?

Reply:
@arianvp This isn't a nixpkgs-maintainer perspective, but FWIW, I've had the best experience with [crate2nix](https://github.com/kolloch/crate2nix), which uses the nixpkgs `buildRustCrate` and seems to work a lot better than carnix. The other tool I've seen people have good experiences with is [naersk](https://github.com/nmattia/naersk) (builds on top of cargo, unlike crate2nix). I think crate2nix output should be suitable for upstreaming to nixpkgs.